simlammps/lammps_process.py

```diff
@@ -0,0 +1,56 @@
+""" LAMMPS Process
+
+This module provides a way to run the lammps process
+"""
+
+import subprocess
+
+
+class LammpsProcess(object):
+    """ Class runs the lammps programm
+
+        Parameters
+        ----------
+        lammps_name :
+            name of lammps executable
+    """
+    def __init__(self, lammps_name="lammps"):
+        self._lammps_name = lammps_name
+        self._returncode = 0
+        self._stderr = ""
+        self._stdout = ""
+
+        # see if lammps can be started
+        try:
+            self.run(" ")
+        except Exception:
+            msg = "Lammps could not be started."
+            if self._returncode == 127:
+                msg += " Lammps executable was not found."
+            else:
+                msg += " stdout/err: " + self._stdout + " " + self._stderr
+            raise Exception(msg)
+
+    def run(self, commands):
+        """Run lammps with a set of commands
+
+        Parameters
+        ----------
+        commands : str
+            set of commands to run
+        """
+
+        proc = subprocess.Popen(
```

Comment:
@itziakos  opening an issue (#14) on this to make sure that i eventually address this.  I briefly looked at it yesterday, but couldn’t see how i could pipe the `commands` string to lammps with `check_call`.  Will look into that more when i get some free time.